dashboard/app/views/devise/sessions/_login.html.haml

```diff
@@ -1,5 +1,9 @@
 - content_for :head do
   %script{src: webpack_asset_path('js/devise/sessions/_login.js')}
+  - if @user_return_to.present?
+    %script
+      :plain
+        window.userReturnTo = "#{@user_return_to}";
```

Comment:
instead of adding this to the window object, could it be passed to `_login.js` using the `getScriptData` pattern?

Reply:
here's an example from the account settings page

![Screenshot 2024-10-31 at 1 26 21 PM](https://github.com/user-attachments/assets/95730ad8-3607-4024-8e1a-b406be92d676)
![Screenshot 2024-10-31 at 1 26 50 PM](https://github.com/user-attachments/assets/dd12327b-5319-4e81-9625-31c0657e87e6)

Reply:
I was able to refactor to this idea- thanks for adding the example!